LICENSE

```diff
@@ -200,3 +200,42 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+
+   --------------------------------------------------
+   SOFTWARE DISTRIBUTED WITH RYA:
+
+   The Apache Rya software includes a number of subcomponents with
+   separate copyright notices and license terms. Your use of the source
+   code for the these subcomponents is subject to the terms and
+   conditions of the following licenses.
+
+   --------------------------------------------------
+   For the extras/rya.jena.sesame/src/main/java/org/apache/rya/jena/jenasesame/ components:
```

Comment:
It may be more sustainable to invert this. For the 3-clause BSD license, we include the following components, the license <follows>. Nit-picky for now.